task1_weatherman/weatherman.py

```diff
@@ -0,0 +1,69 @@
+import os
+import csv
+
+
+
+def get_files_by_year_month(folder_path, target_year, target_month):
```

Comment:
Create a class named. WeatherMachine. Move all these methods in that class. 
Create a main.py file and import the class in it. 
enable file execution in main.py and run the code through `python main.py` call